cool247/asset-management
# add asset.controller

### **Description**
This PR adds functionality to support file uploads in the application. Users can now upload files with the following restrictions:
```

Allowed file types: .jpg, .jpeg, .png, .pdf
Maximum file size: **10MB**
```

### **Changes Made**
```
Added backend validation for file type and size.
Updated frontend to display error messages for invalid files.
Included tests for validation logic.
```

### **How to Test**

> Open the file upload form.
> Upload valid files (e.g., sample.jpg under 10MB) to confirm successful submission.
> Try uploading unsupported files (e.g., test.exe) or files larger than **10MB** and check for appropriate error messages.
